core/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+- Introduce `rustfmt.toml`.
```

Comment:
> we don't need to add a `CHANGELOG.md` entry for this

if I don't add an entry in the `CHANGELOG.md` entry for this, the CI will be all red because it thinks the diffs. I am okay, if that is something you are okay with.